Sources/NetworkService/BodyEncoder/BodyEncoder.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// A protocol for encoding body parameters into an `URLRequest`.
+public protocol BodyEncoder {
+    /**
+     Encodes body parameters (such as JSON, URL-encoded form data, etc.) into the provided `URLRequest`.
+
+     - Parameters:
+        - parameters: An optional `BodyParameters` object containing the data to be encoded.
+        - request: An `inout` `URLRequest` object to which the encoded body will be assigned.
+
```

Comment:
```suggestion
         - parameters: An optional set of parameters containing the data to be encoded.
        - request: A request object to which the encoded body will be assigned.
```
Does it make sense to restrict yourself with concrete types in the documentation?

Reply:
Fixed with https://github.com/ios-course/simple-network-service/pull/13/commits/ca83703f94f9cef74c1a43b45c8d8339217142b1